packages/tree/src/model/node.ts

```diff
@@ -252,7 +255,8 @@ export default class Node {
         parent: this,
         store: this.store,
       })
-      child = new Node(child as TreeNodeOptions)
+      child = reactive(new Node(child as TreeNodeOptions)) as Node
+      (child as Node).initialize()
```

Comment:
Why casting it twice?

Reply:
Because IDE will throw a error, I will change it.

Reply:
I think that was because you cast reactive<Node> to Node